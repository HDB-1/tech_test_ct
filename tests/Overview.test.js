import React from 'react'
import { shallow } from 'enzyme'
import Overview from '../src/components/Overview'


describe("shallow Overview", () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<Overview />)
    })
    it("should render correctly", () => {
        expect(wrapper).toMatchSnapshot();
    })
});

