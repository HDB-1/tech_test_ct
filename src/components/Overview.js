import React, { Component } from 'react';
import Items from './Items';

export default class Overview extends Component {
    render() {
        return (
            <div>
                <p>showing overview component</p>
                <Items allInfo={this.props.info} />
            </div>
        )
    }
}
