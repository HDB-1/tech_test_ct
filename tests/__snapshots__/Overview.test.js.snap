// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow Overview should render correctly 1`] = `
<div>
  <p>
    showing overview component
  </p>
  <Items />
</div>
`;
